anime_downloader/commands/config.py

```diff
@@ -1,6 +1,30 @@
-import click
 import os
+import click
 from tabulate import tabulate
+
+
+def check_running():
+    import psutil
+
+    already_running = False
+    dict_pids = {
+        p.info["pid"]: [p.info["name"], p.info['cmdline']]
+        for p in psutil.process_iter(attrs=["pid", "name", 'cmdline'])
+    }
+
+    if os.getpid() in dict_pids:
+        del dict_pids[os.getpid()]
+
+    for key, value in dict_pids.items():
+        if bool(value[1]):
+            if ('anime config' in ' '.join(value[1]) or
+                    'anime.exe config' in ' '.join(value[1])
```

Comment:
is there not a way just to regex out the words anime config, would remove the or statement.

Reply:
this convo is already discussed in discord, just posting a screenshot here so that i can resolve it
![image](https://user-images.githubusercontent.com/53124886/98954143-2a060080-2506-11eb-82d9-3023555f7f9b.png)